train_segmentation.py

```diff
@@ -616,16 +609,6 @@ def main(params, config_path):
         logging.warning(f'Visualization: expected positive value for ignore_index, got {dontcare_val}.'
                         f'Will be overridden to 255 during visualization only. Problems may occur.')
 
-    # list of GPU devices that are available and unused. If no GPUs, returns empty list
-    gpu_devices_dict = get_device_ids(num_devices,
-                                      max_used_ram_perc=max_used_ram,
-                                      max_used_perc=max_used_perc)
-    logging.info(f'GPUs devices available: {gpu_devices_dict}')
-    num_devices = len(gpu_devices_dict.keys())
-    device = torch.device(f'cuda:{list(gpu_devices_dict.keys())[0]}' if gpu_devices_dict else 'cpu')
-
-    logging.info(f'Creating dataloaders from data in {samples_folder}...\n')
-
```

Comment:
Vic, any reason why you removed this? Were you having trouble with it?

Reply:
moved to model_choice.py fixed too with the frequent bug on HPC of not being able to use GPUs with ids other than 0